aea/helpers/dialogue/base.py

```diff
@@ -183,35 +183,32 @@ def dialogues(self) -> Dict[DialogueLabel, Dialogue]:
         return self._dialogues
 
     @abstractmethod
-    def is_permitted_for_new_dialogue(self, msg: Message, sender: str) -> bool:
+    def is_permitted_for_new_dialogue(self, msg: Message) -> bool:
```

Comment:
You will also have to change this in any derived classes as it is an abstract method. Please search for them but I think it is only fipa which is affected.